lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -16,9 +21,6 @@ module RSpec
       #   #     - when
       #   #     - with
       #   #     - without
-      #   #     - if
-      #   #     - unless
-      #   #     - for
```

Comment:
It's ok to keep this, it's an example of how it can be configured.

Reply:
Re-added.